template/src/screens/IntroScreen/IntroScreen.tsx

```diff
@@ -21,12 +21,13 @@ export function IntroScreen(props: NavigationScreenProps) {
   return (
     <BackgroundImage source={bgImage} resizeMode="cover">
       <StatusBar barStyle="light-content" />
-      <Screen backgroundColor={colors.transparent} paddingTop={60}>
+      <Screen testID="introScreen" backgroundColor={colors.transparent} paddingTop={60}>
```

Comment:
nit: we should probably keep the same casing strategy for testIDs (camelcase vs hyphenated below)

Reply:
totally agree. I found a really odd behavior that on one screen it couldn't find the id for the button when I had the dash :(